KidsIdKit.Shared/Pages/Child/Child.razor

```diff
@@ -19,13 +21,18 @@ else
             <li><a href="/childMedicalNotes/@id">Medical notes</a></li>
         </ul>
     </p>
+    @if (!string.IsNullOrWhiteSpace(CurrentChild.ChildDetails.GivenName) && CurrentChild.ChildDetails.GivenName != "<new>")
+    {
+        <button @onclick="DownloadPdf">Generate Pdf</button>
+    }
 }
 
 @code {
     [Parameter]
     public int id { get; set; }
     Data.Child? CurrentChild;
-
+    private String TemplateString { get; set; }
```

Comment:
This needs to either be type `string?` or be initialized to a non-null value like:

```csharp
private string TemplateString { get; set; } = string.Empty;
```

Reply:
In any case, in C# it is best to avoid the capitalized type names (`String`) vs using .NET types (`string`).

Reply:
Opted for string?